lib/VCMIDirs.cpp

```diff
@@ -132,7 +132,7 @@ bool StartBatchCopyDataProgram(
 		(boost::format("start \"\" /D %1% %2%") % currentPath % (to / exeName));	// Start game in 'currentPath"
 
 	const bfs::path bathFilename = to / "_temp.bat";
-	bfs::ofstream bathFile(bathFilename, bfs::ofstream::trunc | bfs::ofstream::out);
+	std::ofstream bathFile(bathFilename.c_str(), std::ofstream::trunc | std::ofstream::out);
```

Comment:
Are we having a bath here?😄

Reply:
A perfect solution if you're not sure whether to call it `batFile` or `batchFile`!

Reply:
Ïs that StartBatchCopyDataProgram really used somewhere these days?